src/components/menu/__internal__/submenu/submenu.component.js

```diff
@@ -201,6 +205,8 @@ const Submenu = React.forwardRef(
           tabbable={tabbable}
           showDropdownArrow={showDropdownArrow}
           onKeyDown={handleKeyDown}
+          ariaLabel={ariaLabel}
```

Comment:
We already support `ariaHidden` and `ariaLabel` because we're spreading `rest`. I don't think we need to explicitly declare these. I'm not sure we need `ariaHidden`?

Reply:
ariaHidden is so that the aria label and the menu item text aren't both read out. Otherwise a screen reader might go "menu settings settings" if both are set